src/clover/fileparser.py

```diff
@@ -2254,6 +2324,15 @@ def parse_input_files(  # pylint: disable=too-many-locals, too-many-statements
     )
     logger.info("Conversion inputs successfully parsed.")
 
+    # Parse the grids inputs file.
+    (
+        grids,
+    ) = _parse_grid_inputs(
+        inputs_directory_relative_path,
+        logger,
+    )
+    logger.info("Grids inputs successfully parsed.")
+
```

Comment:
Here I am calling def_parse_grid and I am getting an error...  as shown below:
![image](https://user-images.githubusercontent.com/106335872/175827820-f62fec25-8744-4019-aa50-51437ded091e.png)

kindly advise 😄

Reply:
Try switching to just `grids = ` rather than `grids, = `, where you're effectively trying to unpack a tuple, but you've not got a tuple, you've got a single list :smile: